meta-dts-distro/recipes-extended/minio-cli/minio-cli_git.bb

```diff
@@ -9,6 +9,8 @@ inherit go-mod
 LICENSE = "AGPL-3.0-only"
 LIC_FILES_CHKSUM = "file://src/${GO_IMPORT}/LICENSE;md5=eb1e647870add0502f8f010b19de32af"
 
+PV = "1.0+git${SRCPV}"
```

Comment:
https://github.com/minio/mc/releases/tag/RELEASE.2024-06-12T14-34-03Z
Not sure you want to put `RELEASE.2024-06-12T14-34-03Z` or maybe just `2024-06-12T14-34-03Z` there

Reply:
We decided to keep it as it is IIRC 
@macpijan cc